oci-umount.conf

```diff
@@ -6,3 +6,6 @@
 /var/lib/docker-latest/overlay
 /var/lib/docker-latest/devicemapper
 /var/lib/docker-latest/containers
+/var/lib/container/storage/overlay2
+/var/lib/container/storage/overlay
+/var/lib/container/storage/overlay-containers
```

Comment:
What are these files? Who creates these mount points?

Specially the `overlay-containers` one?